### DISCUSSION TITLE
Error No route found for "GET /admin/page/save": Method Not Allowed (Allow: PUT, POST) while savig&publishing Pimcore documents

### DISCUSSION THREAD

**Discussion by pdchaudhary** [pimcore/pimcore#10227] [Q&A / Support]
It happens when I try to add the "update" keyword in WYSIWYG. I get the error "No route found for "GET /admin/page/save": Method Not Allowed (Allow: PUT, POST)".

API is calling in "PUT" request method but still showing the error that from Pimcore admin for document add or edit.

Strangely, if the "update" word is not used in any sentences, then it works fine.

**Comment by pdchaudhary** [comment]
@dvesh3 do you have any idea about this?
Any suggestion for me what should I check?
Thanks in advance!

**Reply by dvesh3** [reply]
@pdchaudhary Hey! I tried to reproduce the issue with steps using "update" keyword in wysiwyg but no luck. Please check logs under `/var/log` for any errors.
